internal/tools/orchestrator/services/addon/addon_handler.go

```diff
@@ -61,21 +61,82 @@ func (a *Addon) AttachAndCreate(params *apistructs.AddonHandlerCreateItem) (*api
 // GetAddonExtention 获取addon的spec，dice.yml信息
 func (a *Addon) GetAddonExtention(params *apistructs.AddonHandlerCreateItem) (*apistructs.AddonExtension, *diceyml.Object, error) {
 	extensionList, err := a.bdl.QueryExtensionVersions(apistructs.ExtensionVersionQueryRequest{Name: params.AddonName, All: true})
+
+	// 类型错误, addon不存在
 	if err != nil || len(extensionList) == 0 {
-		return nil, nil, errors.New("没有匹配的addon信息")
+		err := errors.New("没有匹配的addon信息")
+		logrus.Errorf("cannot find addon : %s", params.AddonName)
+		return nil, nil, err
+	}
+
+	// 规格错误，basic 中间件无法部署在生产环境中
+	if params.Plan == "basic" && params.Workspace == "PROD" {
```

Comment:
There is already logic in Erda that does not allow basic deployments. Please confirm.